armsrc/Makefile

```diff
@@ -24,7 +24,7 @@ APP_CFLAGS = -DWITH_CRC \
 			 -DWITH_FLASH \
 			 -DWITH_SMARTCARD \
 			 -DWITH_HFSNOOP \
-			 -DWITH_LF_SAMYRUN \
+			 -DWITH_HF_COLIN \
```

Comment:
The default standalone mode is SAMYRUN,   easy to have.   Don't change that.

Reply:
It can always be changed,  now that there is the printing of which standalone mode installed..